src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -465,7 +465,8 @@ export const selectPortfolioAccountsCryptoHumanBalancesIncludingStaking =
       return Object.entries(portfolioAccountsCryptoBalances).reduce((acc, [accountId, account]) => {
         acc[accountId] = Object.entries(account).reduce((innerAcc, [assetId, cryptoBalance]) => {
           const asset = assets[assetId]
-          if (asset) innerAcc[assetId] = fromBaseUnit(cryptoBalance, asset.precision)
+          if (asset)
+            innerAcc[assetId] = fromBaseUnit(cryptoBalance, asset.precision, asset.precision)
```

Comment:
unnecessary?

```
// Converts from base unit to a precision/ish number
// - If no displayDecimals are provided, it will use the full precision of the number being converted
// - If displayDecimals are provided, it will use that precision, to return e.g a human, precision, or number stripped to any arbitrary decimal places
// and use the ROUND_DOWN rounding mode
export const fromBaseUnit = (
  value: BigNumber.Value,
  precision: number,
  displayDecimals?: number,
): string => {
```

Reply:
Absolutely - this was diffed before `fromBaseUnit()` was refactored to use the full asset precision. c9e74e2b03540e759eb9a3e4108262f7772892fa